frontend/src/context/datasetStateSlice.ts

```diff
@@ -214,9 +221,22 @@ const fetchHDC = async (
   });
 };
 
+// HDC API expects a parameter which depends on the layer
+// rainfall = rfh; ndvi = vim; blended = rfb
+const getVamParam = (serverLayerName: string): string => {
```

Comment:
curious, why change from a switch to this?

Reply:
you could simply have done the switch on `serverLayerName`

Reply:
The `getVamParam` function was there before but was initializing inside `loadAdminBoundaryDataset` function so I moved it outside and also the switch case was initializing with `switch(true)`, see below function. I found it for this specific occasion to be more clear using if statements. If we are not ok with it I can convert it back to where it was.